src/components/MessageListItem.react.js

```diff
@@ -2,6 +2,15 @@ import React from 'react';
 import { PropTypes } from 'prop-types';
 import { Map, Marker, Popup, TileLayer } from 'react-leaflet';
 import Linkify from 'react-linkify';
+import {
+  Table,
+  TableBody,
+  TableHeader,
+  TableHeaderColumn,
+  TableRow,
+  TableRowColumn,
+} from 'material-ui/Table';
```

Comment:
Sorry I missed that discussion. We dint use any other material UI modules? So do we let it be till we decide on something?